### PR TITLE
DT-476:

### PR DESCRIPTION
**Pull request by mhutton86**
- Fix localization problem with CSV reader
- Fix defect where V1 TVP checks were preventing some V2 TVP checks from running

### CODE REVIEW

File: Forms/CheckResultsForm.cs [hunk 1]
```diff
@@ -617,11 +648,12 @@ private void ExecuteChecksAndStoreResults(string content, List<CheckAndFixItem>
                                 // check the verse with verse checks
                                 ExecuteChecksAndStoreResults(verseText, ChecksByScope[CheckScope.VERSE], currBookNum, currChapterNum, currVerseNum);
                             }
-                            catch (ArgumentException)
+                            catch (ArgumentException ae)
                             {
-                                // arg exceptions occur when verses are missing, 
+                                // arg exceptions occur when verses are missing,
                                 // which they can be for given translations (ignore and move on)
                                 // TODO spit out warning
```
Comment (hndykstra) [inline Forms/CheckResultsForm.cs]: TODO

File: Text/BookUtil.cs [hunk 2]
```diff
@@ -40,24 +40,34 @@ public class BookUtil
 
         static BookUtil()
         {
+            // ready in our book integer ID to canonincal ID map
```
Comment (hndykstra) [inline Text/BookUtil.cs]: canonical

File: TranslationValidationPlugin.cs [hunk 3]
```diff
@@ -65,6 +66,11 @@ public void Run(IHost host, string activeProjectName)
                     {
                         try
                         {
+                            //CultureInfo englishUSCulture = new CultureInfo("en-US");
```
Comment (hndykstra) [inline TranslationValidationPlugin.cs]: some reason to leave this in? if so, a comment would be helpful.

**Review by hndykstra — APPROVED** [review]

File: TranslationValidationPlugin.cs (hunk 3 above)
Comment (mhutton86) [reply TranslationValidationPlugin.cs]: Nope. Removed.

File: Text/BookUtil.cs (hunk 2 above)
Comment (mhutton86) [reply Text/BookUtil.cs]: Fixed.

File: Forms/CheckResultsForm.cs (hunk 1 above)
Comment (mhutton86) [reply Forms/CheckResultsForm.cs]: Not needed. Removed.

**Pull request merged**